src/PlayerOptions.h

```diff
@@ -398,6 +400,12 @@ class PlayerOptions
 	/** @brief The Visual Delay additionally applied on a per-player basis in ms. */
 	float	m_fVisualDelay;
 
+	/** @brief The TimingWindow that can be disabled. Valid values are only W1-W5.
+     *  Other values are ignore.
+	 *  We use a set instead of unordered_set because we want the ordering so that 
+	 *  the generated player options string is consistent. */
+	std::set<TimingWindow> m_twDisabledWindows;
```

Comment:
I think we should use an `std::bitset` here. This structure is used on every update, so better make it efficient.

Reply:
Done! I feel a little iffy about the implicit conversion from `TimingWindow` -> `int`/`size_t` when indexing into the `bitset` but I think it's okay. The alternative was to use an `unordered_map` but `bitset` already has the same functionality we're looking for anyways.